automations/python/shared/actions.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+
+def write_to_github_output(lines: list[str]) -> None:
+    """
+    Write a list of strings to both stdout and the GITHUB_OUTPUT environment variable.
```

Comment:
Why do we need both? What is `stdout` output used for?

Reply:
I like writing to `stdout` so that it's printed in the workflow logs and helps with debugging.